lib/licensee/matchers/cargo.rb

```diff
@@ -0,0 +1,16 @@
+module Licensee
+  module Matchers
+    class Cargo < Licensee::Matchers::Package
+      LICENSE_REGEX = /
+        ^\s*[\'\"]?license[\'\"]?\s*=\s*[\'\"]([a-z\-0-9\._]+)[\'\"]\s*
```

Comment:
You may also want to allow internal parens.  They are not yet supported by crates.io (withoutboats/license-exprs#3, and I'm documenting that limitation in rust-lang/cargo#4898), but hopefully they'll be supported at some point.  And I don't see a problem with allowing them in this regex now.

Reply:
> You may also want to allow internal parens…

I've filed support for this in #261, although Licensee will still return `other` for those cases.